packages/server/graphql/mutations/helpers/resolveDowngradeToStarter.ts

```diff
@@ -48,7 +52,9 @@ const resolveDowngradeToStarter = async (
     orgName: org.name,
     oldTier: 'team',
     newTier: 'starter',
-    billingLeaderEmail: user.email
+    billingLeaderEmail: user.email,
```

Comment:
Maybe we don't need to track this property at all? Looks like it matches email, as we take this from `viewerId`.
@tghanken Do we need that property? 

```
analytics.track({
  userId: 'local|jQD5bQyFEs',
  event: 'Organization Downgraded',
  properties: {
    billingLeaderEmail: 'igor@example.com',
    domain: 'example.com',
    email: 'igor@example.com',
    newTier: 'starter',
    oldTier: 'team',
    orgId: 'jXKJCeJ9ok',
    orgName: 'archive test'
  }
});
```

Reply:
@igorlesnenko where does the email field get appended to the properties object?

Reply:
@nickoferrall Looks like it is coming from this kind of calls: https://github.com/ParabolInc/parabol/blob/3c90a6facc7691ec482fee1a9e5504508eb49c18/packages/server/graphql/mutations/helpers/bootstrapNewUser.ts#L33